scripts/merge_tables.R

```diff
@@ -21,9 +25,10 @@ script <- sub("--file=", "", basename(commandArgs(trailingOnly=FALSE)[4]))
 # Build description message
 description <- "Merge miRNAs quantification tables.\n"
 author <- "Author: Paula Iborra, Biozentrum, University of Basel"
-version <- "Version: 1.0.0 (JUN-2019)"
-requirements <- "Requires: optparse"
-msg <- paste(description, author, version, requirements, sep="\n")
+mantainer <- "Refactor and documentation: Iris Mestres"
```

Comment:
You could put this as just:

`maintainer: Iris Mestres <your@email.address>`

without the `Refactor and documentation`.

And while you are at it, perhaps change this one as well:

`author: Zavolan Lab <zavolab-biozentrum@unibas.ch>`

If you prefer, you could also set both to `zavolab-biozentrum@unibas.ch`.